src/fenicsxconcrete/finite_element_problem/concrete_thermo_mechanical.py

```diff
@@ -106,8 +110,12 @@ def default_parameters() -> tuple[Experiment, dict[str, pint.Quantity]]:
             "ft_inf": 467000 * ureg(""),
             "a_ft": 1.0 * ureg(""),
             "evolution_ft": "True" * ureg(""),
+            "dt": 1.0 * ureg("s"),
         }
-        default_parameters["E_act"] = 5653.0 * default_parameters["igc"] * ureg("J/mol")
+        default_parameters["E_act"] = (
+            5653.0 * default_parameters["igc"].magnitude * ureg("J/mol")
```

Comment:
Don't know why this format is chosen, but $5653.0$ is $E_a/R$ and has unit $K$, because $E_a$ has unit $J/\mathrm{mol}$. The following expression should yield the correct unit, however it is not very explicit.
```suggestion
            5653.0 * ureg("K") * default_parameters["igc"] 
```
Alternatively
```suggestion
            5653.0 * default_parameters["igc"].to_base_units().magnitute * ureg("J/mol")
```